### PR TITLE
Consider captures as a requirement for pro players

### LINKED ISSUE #525: Take captures into consideration for "pro" players.
Currently, pro players are those who have a score more than 10,000 and a K/D of more than 1.5. The captures also need to be added as a criteria in pertaining a pro player.

### PR DESCRIPTION
**Pull request by MinetestSam**
Continuation of #527 as I deleted my repo by mistake.

Closes #525

### CODE REVIEW

**Comment by fjmng** [comment]
The formspec to unlock the pro section of the chest and the random messages should be updated

**Comment by MinetestSam** [comment]
> The formspec to unlock the pro section of the chest and the random messages should be updated

I think I did update the formspec? As for the random message, I would add that once the value of captures has been decided

**Comment by MinetestSam** [comment]
Hello, I am busy with real life stuff and won't be working on CTF things anymore. So this PR won't have any further updates. The devs can close it if they feel like it.

**Comment by LoneWolfHT** [comment]
Fixed conflicts

**Comment by Avyukt1** [comment]
I think its a good idea to make captures as requirements for pro players because a player called minetest has 4.3 k/d but **0 attempts!**

**Review by moreavy — APPROVED** [review]

File: mods/ctf/ctf_stats/init.lua [hunk 1]
```diff
@@ -207,7 +207,7 @@ end
 function ctf_stats.is_pro(name)
 	local stats = ctf_stats.player(name)
 	local kd = stats.kills / (stats.deaths == 0 and 1 or stats.deaths)
-	return stats.score >= 10000 and kd >= 1.5
+	return stats.score >= 10000 and kd >= 1.5 and stats.captures >= 10
```
Comment (moreavy) [inline mods/ctf/ctf_stats/init.lua:210]:
```suggestion
	return (stats.score >= 10000) and (kd >= 1.5) and (stats.captures >= 10)
```

Maybe this is safer.

Comment (LoneWolfHT) [reply mods/ctf/ctf_stats/init.lua:210]: Not needed

**Pull request merged**